### ISSUE TITLE
Long Polling

### ISSUE THREAD

**Issue by MichalKononenko** [TopChef/TopChefClient#14]
**TopChef Project Issue Request**

**_Issue Type**_
My issue is a
- [ ] bug (something broke)
- [x] feature request (I'd like this in the API)
- [ ] general inquiry (I have a question about the system)

**_Description**_

Please provide a detailed description of your bug, feature, or
question here

Long polling would give much faster response time than the current polling approach. It may require some doing on the API as well.

Basically, do [this](https://youtu.be/tdIIJuPh3SI?t=8147). Shoutout to @miguelgrinberg!

EDIT: [There goes my hero](https://www.youtube.com/watch?v=87bcPisyTHs)

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/203687696940382/203687704139609)
